sdk/dotnet/Helm/ChartBase.cs

```diff
@@ -84,6 +87,9 @@ protected ChartBase(string releaseName, Union<ChartArgs, LocalChartArgs> args, C
                     var data = JsonSerializer.Serialize(cfgBase.Values);
                     File.WriteAllText(overrides, data);
 
+                    // Get helm version
+                    var helmv3 = IsHelmV3();
```

Comment:
Nit: is there a reason to have a variable declared for this 30 lines before it's used? If not, I'd just inline the call into the `if`.

Reply:
good point, will fix